src/components/PeopleFilters.tsx

```diff
@@ -1,18 +1,81 @@
+import { useSearchParams } from 'react-router-dom';
+import { SexFilters } from '../types/SexFilters';
+import classNames from 'classnames';
+import { ChangeEvent, useMemo } from 'react';
+
 export const PeopleFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const sexFilter = searchParams.get('sex');
+  const centuriesArray = searchParams.getAll('centuries');
+
+  const centuriesData = useMemo(() => [16, 17, 18, 19, 20], []);
```

Comment:
```suggestion
  const centuriesData = [16, 17, 18, 19, 20];
```
it's a static value, so you can just define it _above_ component in this file